web/src/features/panels/zone/DisplayByEmissionToggle.tsx

```diff
@@ -1,21 +1,18 @@
-import ToggleButton from 'components/ToggleButton';
-import { useAtom, useAtomValue } from 'jotai';
+import ToggleButton, { ToggleButtonOptions } from 'components/ToggleButton';
+import { useAtom } from 'jotai';
 import type { ReactElement } from 'react';
 import trackEvent from 'utils/analytics';
 import { LeftPanelToggleOptions, TrackEvent } from 'utils/constants';
-import { displayByEmissionsAtom, isConsumptionAtom } from 'utils/state/atoms';
+import { displayByEmissionsAtom } from 'utils/state/atoms';
```

Comment:
Ooh are we ready to say goodbye to production for real?

Reply:
Since we changed it to electricity it don't make much sense to switch between "Electricity" and "Production".